examples/iot_lock.md

```diff
@@ -22,6 +22,6 @@ There are wide applications for Tupelo for IoT because it is efficient,
 fast and even includes the possibility of devices being able to check transaction
 validity when they are offline.  A small example of offline functionality
 [can be found here](https://github.com/quorumcontrol/offliner) but hop into our
-developer chat or ask for more information if you have any questions.
+[developer chat](https://t.me/joinchat/IhpojEWjbW9Y7_H81Y7rAA) or ask for more information if you have any questions.
```

Comment:
ask for more information where? :)

Reply:
There is a form on most pages... I can add it to this one or just remove that reference... hmm...

Reply:
Added the info form on the page.  If someone gets this far we REALLY want to know who they are so seemed like a decent choice.  May look at slightly better styling for the form but that can be later.